groundskeeper

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Not sure how you feel about this.  It makes things a bit easier on macOS which ships ancient bash by default.

Reply:
👍